include/stdexec/__detail/__let.hpp

```diff
@@ -639,5 +657,19 @@ namespace stdexec {
   inline constexpr let_stopped_t let_stopped{};
 
   template <class _SetTag>
-  struct __sexpr_impl<__let::__let_t<_SetTag>> : __let::__let_impl<_SetTag> { };
+  struct __sexpr_impl<__let::__let_tag<_SetTag>> : __let::__let_impl<_SetTag> { };
+
+  template <class _SetTag>
+  struct __sexpr_impl<__let::__let_t<_SetTag>> : __sexpr_defaults {
+    static constexpr auto get_attrs = []<class _Child>(__ignore, const _Child& __child) noexcept {
+      // BUGBUG:
+      return stdexec::get_env(__child);
+      //return __attrs<__let_t<_SetTag>, _Child, _Fun>{};
+    };
+
+    static constexpr auto get_completion_signatures =
+      []<class _Sender, class... _Env>(_Sender&&, const _Env&...) noexcept
+      -> __completion_signatures_of_t<transform_sender_result_t<_Sender, _Env...>, _Env...> {
+    };
```

Comment:
why are these needed?

Reply:
It's definitely necessary to have the `get_completion_signatures`, this is the default implementation thereof:

```c++
static constexpr auto get_completion_signatures =
  []<class _Sender>(_Sender&&, auto&&...) noexcept {
    static_assert(
      __mnever<tag_of_t<_Sender>>,
      "No customization of get_completion_signatures for this sender tag type.");
  };
  ```

I can yank `get_attrs` if you're happy to lose the comment.